integration/test_collection.py

```diff
@@ -68,8 +68,23 @@ def test_create_and_delete(client: weaviate.Client):
     assert not client.collection.exists(name)
 
 
-def test_create_get_and_delete_with_supported_generic(client: weaviate.Client):
-    name = "TestCreateGetAndDeleteGeneric"
+def test_create_and_delete_with_dict_generic(client: weaviate.Client):
+    name = "TestCreateAndDeleteDictGeneric"
+    collection_config = CollectionConfig(
+        name=name,
+        properties=[Property(name="Name", data_type=DataType.TEXT)],
+        vectorizer=Vectorizer.NONE,
+    )
+    client.collection.create(collection_config)
+
+    client.collection.get(name, Dict[str, str])
+    assert client.collection.exists(name)
+    client.collection.delete(name)
+    assert not client.collection.exists(name)
```

Comment:
this tests the create call, not the get call with the Dict[]

Reply:
Ah gotcha, `.create()` uses `.get()` under-the-hood but the test doesn't know that!

Reply:
but still the difference is in the colelction that gets returned - so I would expect that the collection is tested - eg an insert + a return

Reply:
> This is a good time to note that test_collection.py should be split up into collection/data, collection/grpc, etc

agreed. Follow-up PR? :)